File: season1/2.py
def average(arr, s, c):
    if len(arr) == c:
        return s / c
    else:
        s += arr[c]
        c += 1
        return average(arr, s, c)


array = input("please enter the numbers :").split()
array = [int(item) for item in array]
print(average(array, 0, 0))

File: season2/2.cpp
#include <iostream>
using namespace std;
int c(int n,int r){
    if( n==1 || n==2)
        return 1;
    if ((n-r)>r)
        r=n-r;
    int arr[2*(n-r+1)];
    arr[0]=1;
    for (int i = 1; i <= n-r; i++)
    {
       arr[i]=arr[i-1]*(r+i);
    }
    for (int i = 1; i <= n-r; i++)
    {
       arr[n-r+i]=arr[n-r+i-1]/i;
    } 
    //     for (int i = 0; i <=2*(n-r); i++)
    // {
    //   cout<<arr[i]<<endl;
    // } 
    return arr[2*(n-r)];
}   
int main(){
    cout <<"enter two numbers(n and r):";
    int n;
    int r;
    cin >> n>>r;
    cout<<c(n,r);
}